requirements_test.txt

```diff
@@ -3,7 +3,8 @@ docopt
 psutil
 requests
 ansible
-ansible-core
+ansible-core==2.16; python_version > "3.11"
```

Comment:
will it pick ansible-core2.16 if python_version = 3.11?

Reply:
good catch. fixed.